src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Equality.txt

```diff
@@ -0,0 +1,71 @@
+﻿>> 1 = 1
+true
+
+>> 1 <> 1
+false
+
+>> 1 = 2
+false
+
+>> 1 <> 2
+true
+
+>> true = true
+true
+
+>> true = false
+false
+
+>> true <> true
+false
+
+>> true <> false
+true
+
+>> 1.0 = 1.0
+true
+
+>> 1.0 <> 1.0
+false
+
+>> 1.0 <> 1.1
+true
+
+>> 1.0 = 1.1
+false
+
+>> "foo" = "foo"
+true
+
+>> "foo" <> "foo"
+false
+
+>> "foo" = "bar"
+false
+
+>> "foo" <> "bar"
+true
+
+>> Date(2022,2,17) = Date(2022,2,17)
+true
+
+>> Date(2022,2,17) <> Date(2022,2,17)
+false
+
+>> Date(2022,2,17) = Date(2017,4,27)
+false
+
+>> Date(2022,2,17) <> Date(2017,4,27)
```

Comment:
Some Time test as well?

Reply:
Not sure if we have Guid or color functions yet, but that'd be good as well if they exist

Reply:
I just added Time tests
I'm not sure of the syntax for Guid or colors

Reply:
There is a Guid function ([https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/functions/function-guid](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/functions/function-guid) ) to return guids. 

But it's not impl in interpreter - that could be a separate PR.

Reply:
I was debugging ColorValue and it seems not implemented too - another PR as well I guess

Reply:
![image](https://user-images.githubusercontent.com/69138830/154579017-bdef3128-840f-408b-b937-9df182a1a99e.png)